### ISSUE TITLE
Contrasto insufficiente del testo nella riga di warning

### ISSUE THREAD

**Issue by martalucivero** [italia/graph-italia#134]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [x] Ho verificato e non esiste

### Cosa

All'interno della modale è presente una riga evidenziata tramite testo di colore giallo chiaro.
Il contrasto tra il testo e lo sfondo non raggiunge il rapporto minimo richiesto dalle WCAG, rendendo il contenuto difficilmente percepibile.

### Perché

WCAG 1.4.3 – Contrast (Minimum) (Livello AA): il testo deve avere un rapporto di contrasto di almeno 4,5:1, oppure 3:1 nel caso di testo grande.

### Contesto

<img width="580" height="349" alt="Image" src="https://github.com/user-attachments/assets/23a1ba87-e9db-4b20-8184-799c2c92481a" />
